src/main/webapp/groupsPage.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
Why do we need this page when we have the Soy template? See if my comments on https://github.com/googleinterns/step123-2020/pull/10 are relevant here.